_posts/2025-07-21-vox-pupuli-has-grown-up.md

```diff
@@ -0,0 +1,75 @@
+There isn't a formal date when puppet-community was founded.
+But we know the people that were involved in the early days:
+
+* [daenney](https://github.com/daenney#-maintainer)
+* [Mina](https://igalic.co/)
+* [ffrank](https://felix-frank.net/)
+* [nibalizer](https://spencerkrum.com/)
+
+By a lucky accident I had the chance to attend a puppet practitioner training in 2013.
+This was organized by [Netways](https://netways.de/).
+I joined puppet-community in 2014 and was mostly lurking around.
+I became more active over time and nibalizer convinced me to make my first talk at a conference, [Config Manangement Camp Berlin, 2016](https://cfgmgmtcamp.org/berlin2016/).
+I gave an ignite talk about Vox Pupuli, and the slides are [still around](https://bastelfreak.de/cfgcamp/#1).
+
+People that had to endure my more recent talks might notice that I still use the same slide styleset as almost a decade ago, which I got from Mina back then.
+This is of course a homage to Mina, I'm not just lazy.
+
+My ignite talk went well and I was happy to meet so so many people back then. It was a very friendly event with a welcoming community.
+
+nibalizer did the second keynote in at Config Manangement Camp Berlin.
+He went on stage with a neck tie.
+
+<img alt="the awesome, the one and only, nibalizer!" src="{{ site.url }}{{ site.baseurl }}/static/images/IMG_9128.JPG" width="50%" height="50%" />
+
+The message back then: Open source is serious business.
+And in a serious business you sometimes wear ties.
+
+Almost a decade ago he already realized how important open source communities are and how much value they produce as a collective, and how much every individual contributor benefits from it.
+
+### Fast forward, July 2025
+
+In the past months, I had the pleasure to organize the first Vox Pupuli conference, [VoxConf](https://voxpupuli.org/voxconf/).
+This was the first conference for many community members.
+The goal was to give them the same welcoming feeling as I had on my first conference.
+
+The event was hosted by Netways, the same company that organized my puppet training back then.
+The recommended VoxConf hotel was the Park Inn hotel, where the training took place.
+My trainer back then was Lennart Betz, and I'm more than happy that we could convince him to not only give a talk at VoxConf, but also provide an awesome city tour the day before.
+The picture above was taken by Dirk Götz, who works for Netways and is deeply involved in the Foreman ecosystem.
+Vox Pupuli and Foreman are closely collaborating since years, so it made sense to me to host our event alongside the Foreman birthday party.
+ffrank wasn't much active in the community for some time, so I was very happy to see him again at VoxConf.
+And of course I reused again the slide styleset from Mina.
+
+And of course this had to happen:
+
+<img alt="bastelfreak with a neck tie" src="{{ site.url }}{{ site.baseurl }}/static/images/pxl_20250717_065650108.jpg" width="50%" height="50%" />
```

Comment:
also 404

Reply:
mhm how did you check this? `bundle exec jekyll serve -w --config _config.yml,_config-dev.yml` renders it fine for me

Reply:
ah... i checked the github branch preview